docs/01_overview/01_framework_and_toolchain.md

```diff
@@ -45,5 +45,7 @@ You can read detailed documentation about `keosd` [here](http://docs.eosnetwork.
 ## CDT
 CDT is a toolchain for WebAssembly (Wasm) and a set of tools to facilitate contract writing for the Antelope framework. In addition to being a general-purpose WebAssembly toolchain, Antelope-specific optimizations are available to support building Antelope smart contracts. This new toolchain is built around Clang 7, which means that CDT has most of the current optimizations and analyses from LLVM.
 
+You can read more about `CDT` [here](https://github.com/AntelopeIO/cdt).
```

Comment:
To prevent site hopping into github back and forth, we can just link directly to the CDT docs within the portal: https://docs.eosnetwork.com/cdt/latest/